File: Daily challenge/Time Needed to Buy Tickets.py
class Solution(object):
    def timeRequiredToBuy(self, tickets, k):
        count = 0
        while True:
            for i in range(len(tickets)):
                if tickets[i] > 0:
                    tickets[i] -= 1
                    count += 1

                if tickets[k] == 0:
                    return count
        

File: Daily challenge/Time Needed to Buy Tickets.cpp
class Solution 
{
public:
    int timeRequiredToBuy(vector<int>& tickets, int k) 
    {
        int count = 0;
        while(tickets[k])
        {
            for(int i = 0; i < tickets.size(); i++)
            {
                if(tickets[i] > 0)
                {
                    tickets[i]--;
                    count++;
                }
                if(tickets[k] == 0)
                {
                    return count;
                }
            }
        }
        return count;
    }
};